From 5ebefa1ef61648e977f88fa722bba98b7f73dcfc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20Iwanicki?= <michal.iwanicki@3mdeb.com>
Date: Fri, 13 Mar 2026 10:14:26 +0100
Subject: [PATCH] slaunch: psp: print base_addr
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [debug]
Signed-off-by: Michał Iwanicki <michal.iwanicki@3mdeb.com>
---
 grub-core/loader/slaunch/psp.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/grub-core/loader/slaunch/psp.c b/grub-core/loader/slaunch/psp.c
index f30d79063402..015b13f90b93 100644
--- a/grub-core/loader/slaunch/psp.c
+++ b/grub-core/loader/slaunch/psp.c
@@ -285,6 +285,8 @@ init_drtm_interface (grub_addr_t base_addr, psp_version_t version)
       return grub_error (GRUB_ERR_BAD_DEVICE, N_("DRTM: Unrecognized PSP version %d\n"), version);
     }

+    grub_dprintf ("slaunch", "base_addr: %08x\n", base_addr);
+
   return GRUB_ERR_NONE;
 }

--
2.52.0
